assets/src/edit-story/elements/media/edit.js

```diff
@@ -134,8 +135,35 @@ function MediaEdit({ element, box }) {
 
   const isImage = 'image' === type;
   const isVideo = 'video' === type;
+
+  const wheelHandler = useCallback(
+    (evt) => {
+      let newScale = scale || 100;
+      if (event.deltaY > 0) {
+        newScale += 10;
+      } else {
+        newScale -= 10;
+      }
+      setProperties({ scale: newScale });
+      evt.preventDefault();
+    },
+    [setProperties, scale]
+  );
+
+  useEffect(() => {
+    const editEl = editElementRef.current;
+    editEl.addEventListener('wheel', wheelHandler, {
+      passive: false,
+    });
+    return () => {
+      editEl.removeEventListener('wheel', wheelHandler, {
+        passive: false,
+      });
+    };
+  });
+
   return (
-    <Element>
+    <Element ref={editElementRef}>
```

Comment:
I wonder if the event listener should work on the faded media. Otherwise, the user would always have to scroll within the visibly displayed part of the media only. Thoughts?